capstone-ui/src/Components/App/App.jsx

```diff
@@ -27,6 +27,8 @@ export default function App() {
     updateUser(null);
   };
 
+  const [selectedCandidates, setSelectedCandidates] = useState([])
```

Comment:
I think it makes sense to have `selectedCandidates` state in top level for use between both pages. However, I don't think it makes sense here as this is the root level of your app. Is it possible to instead have an `InteractiveMap` component that has routes setup for `PresidentialRaces` and `PresidentStatePage`?
For example:
```
App
  - Routes
    - Login
    - PresidentialRacesBasePage
      - Routes
        - PresidentialRaces
        - PresidentStatePage 
```

In addition, this issue can also be avoided by having `PresidentialRaces` AND `PresidentStatePage` as one page where selecting can either zoom in or highlight.

Reply:
I want to take a look at having them both in the same page with it being highlighted.